youtube_dl/extractor/mdr.py

```diff
@@ -128,12 +140,19 @@ def _real_extract(self, url):
 
                     f = {
                         'url': video_url,
-                        'format_id': '%s-%d' % (media_type, vbr or abr),
-                        'filesize': filesize,
```

Comment:
Revert.

Reply:
If I revert this, then it’ll crash again, because neither `vbr` nor `abr` are defined.